sensor_msgs/PointCloud2 pointcloud
---
shape_msgs/Mesh mesh